app/Http/Controllers/Dashboard/ComponentController.php

```diff
@@ -304,7 +307,9 @@ public function updateComponentGroupAction(ComponentGroup $group)
                 $group,
                 Binput::get('name'),
                 $group->order,
-                Binput::get('collapsed')
+                Binput::get('collapsed'),
+                Binput::get('visible'),
+                app(Guard::class)->user()->getKey()
```

Comment:
Rather than passing the "key" of the user, we should be passing the user object.